flight_planning/v1/flight_planning.yaml

```diff
@@ -0,0 +1,371 @@
+    attempting to plan, update, and close flight plans.
+    
+    The client may also (with a separate authorization scope) act as the test director to determine the status of the
+    USS's system under test, and request that the test area be cleared of any dangling flight plans.
+
+components:
+  securitySchemes:
+    Authority:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: https://auth.example.com/oauth/token
+          scopes:
+            interuss.flight_planning.direct_automated_test: |-
+              Client may determine the test-readiness of the USS and instruct the USS to perform various actions to
+              prepare for tests involving flight planning (acting as test director).
+            interuss.flight_planning.plan: |-
+              Client may act as a user attempting to plan a flight, modify an existing flight plan, or close an existing
+              flight plan.
+      description: |-
+        Authorization from, or on behalf of, an authorization authority, augmenting standard strategic conflict detection or other similar authorization for the purpose of automated testing.
+
+  schemas:
+    StatusResponse:
+      type: object
+      required:
+        - status
+      properties:
+        status:
+          description: >-
+            The status of the USS automated testing interface.
+
+            - `Starting`: the USS is starting and the automated test driver should wait before sending requests.
+
+            - `Ready`: the USS is ready to receive test requests.
+          type: string
+          enum: [Starting, Ready]
+          example: Ready
+        system_version:
+          description: |-
+            Arbitrary string representing the version of the USS system to be tested.
+          type: string
+          example: v0.0.1-445ad3
+        api_name:
+          description: |-
+            Indication of the API implemented at this URL.  Must be "Flight Planning Automated Testing Interface".
+          type: string
+          example: Flight Planning Automated Testing Interface
+        api_version:
+          description: |-
+            Indication of the API version implemented at this URL.  Must be "v0.1.0" when implementing this version of the API.
+          type: string
+          example: v0.1.0
+
+    FlightPlan:
+      description: >-
+        Details of user's intent to create or modify a flight plan.
+      required:
+        - basic_information
+      type: object
+      properties:
+        basic_information:
+          $ref: '#/components/schemas/BasicFlightPlanInformation'
+        astm_f3548_21:
+          $ref: './scd.yaml#/components/schemas/ASTMF354821OpIntentInformation'
+        uspace_flight_authorisation:
+          $ref: './uspace.yaml#/components/schemas/FlightAuthorisationData'
+        au:
+          $ref: './au.yaml#/components/schemas/AUFlightDetails'
+        additional_information:
```

Comment:
Would it make sense using `additionalProperties: ...`  here?

Reply:
`additionalProperties` specifies the expected schema of the values of a dict for keys not otherwise defined.  Since I don't think we know what that schema would be, and because it would likely be different for different jurisdictions/needs, I don't think we can specify anything here.